packages/material-ui/src/styles/createMuiTheme.d.ts

```diff
@@ -31,6 +32,7 @@ export interface Theme {
   shape: Shape;
   breakpoints: Breakpoints;
   direction: Direction;
+  dateIOAdapter?: any;
```

Comment:
Should we can it `date`?

Reply:
Just `dateAdapter`? Will it be clear what does it mean?

Reply:
cc @mui-org/core.